iothub/service/src/Registry/Models/ExportImportDevice.cs

```diff
@@ -128,7 +94,7 @@ public ExportImportDevice(Device device, ImportMode importmode)
         /// access keys are auto-generated.
         /// </remarks>
         [JsonPropertyName("authentication")]
-        public AuthenticationMechanism Authentication { get; set; }
+        public AuthenticationMechanism Authentication { get; set; } = new();
```

Comment:
I noticed that we do this in a bunch of places; any reason why we want this default to empty object and not null?

Reply:
To avoid NRE for users when they explore an object.